src/checkers/inference/solver/backend/encoder/ConstraintEncoderCoordinator.java

```diff
@@ -1,29 +1,31 @@
 package checkers.inference.solver.backend.encoder;
 
 import checkers.inference.model.BinaryConstraint;
-import checkers.inference.model.CombineConstraint;
 import checkers.inference.model.ConstantSlot;
 import checkers.inference.model.ExistentialConstraint;
 import checkers.inference.model.PreferenceConstraint;
+import checkers.inference.model.TernaryConstraint;
 import checkers.inference.model.VariableSlot;
 import checkers.inference.solver.backend.encoder.binary.BinaryConstraintEncoder;
-import checkers.inference.solver.backend.encoder.combine.CombineConstraintEncoder;
 import checkers.inference.solver.backend.encoder.existential.ExistentialConstraintEncoder;
 import checkers.inference.solver.backend.encoder.preference.PreferenceConstraintEncoder;
+import checkers.inference.solver.backend.encoder.ternary.TernaryConstraintEncoder;
 
 /**
- * A coordinator class that has the coordinating logic how each encoder encodes its supported constraint.
+ * A coordinator class that has the coordinating logic how each encoder encodes its supported
+ * constraint.
  * <p>
  * Dispatching example: this class dispatches the encoding of {@link BinaryConstraint} to the
  * corresponding encodeXXX_YYY() method in {@link BinaryConstraintEncoder} depending on the
- * {@link SlotSlotCombo} of {@link BinaryConstraint} that the encoder encodes.
+ * {@link SlotSlotCombo} of {@link BinaryConstraint} that the encoder encodes. Similarly for
```

Comment:
It would read the same, "this class also dispatches the encoding of TernaryConstraint to the coresponding encodeXXX_YYY() method in TernaryConstraintEncoder depending on the SlotSlotCombo of TernaryConstraint that the encoder encodes". Do you want this duplication?

Reply:
You're right.

Reply:
Yes, we don't need this. I though it was code example.